content/participants/ttjc@priyanshps/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Priyansh shukla
+type: "participant"
+name: "Priyansh Shukla"
+portfolio: "coming_soon..."
+github-link: "https://github.com/priyanshps"
+linkedin-link: "www.linkedin.com/in/priyanshps"
+looking-for: "Job"
+description: "node | mongodb | mysql | php"
+---
+
+Hi! I am Priyansh Shukla, B.Tech CSE final year student at RGPV university,let me walk you through my intro, projects and blogs.
+
+### introduction
+
+I am Tech Lover. Love to explore technology.I'm a great learner 
+
+## why are you taking this challenge?
+
+Looking for an organization where I can use the skills I have learnt .
+
+
+## something interesting about you?
+
+- I can convert thoughts into project will drinking Tea.
```

Comment:
do you mean `while drinking tea`

Reply:
Sorry tanay sir application is now updated. will is now updated to while